L1Trigger/Phase2L1ParticleFlow/python/l1ctLayer1_cff.py

```diff
@@ -8,11 +8,10 @@
 from L1Trigger.Phase2L1ParticleFlow.l1tPFClustersFromHGC3DClusters_cfi import l1tPFClustersFromHGC3DClusters
 
 from L1Trigger.Phase2L1ParticleFlow.l1TkEgAlgoEmulator_cfi import tkEgAlgoParameters,tkEgSorterParameters
-from L1Trigger.VertexFinder.l1tVertexProducer_cfi import l1tVertexProducer
 
 from L1Trigger.Phase2L1ParticleFlow.mlAssociation_cfi import NNVtxAssociationPSet
 switchOnNNAssoc = cms.bool(False)
-# switchOnNNAssoc = cms.bool(True)
+switchOnNNAssoc = cms.bool(True)
```

Comment:
Sorry - I left that on by mistake after running some tests. Can remove on the next commit.